Desktop/ruby_test/calender.rb

```diff
@@ -0,0 +1,71 @@
+require 'date'
+require 'optparse'
+
+opt = OptionParser.new
+opt.on('-m') { |v| }
+opt.parse!(ARGV)
+num = ARGV.to_a
```

Comment:
num[0]のように配列のインデックスで参照すると何を意味するのかわかりづらいので、
具体的な意味をもつ変数に入れるとよいと思います。

```suggestion
month = ARGV.to_a[0]
```